packages/stark-core/src/http/services/http.service.ts

```diff
@@ -313,6 +311,17 @@ export class StarkHttpServiceImpl<P extends StarkResource> implements StarkHttpS
 			httpResponse$ = this.addRetryLogic(httpResponse$, retryCount);
 		}
 
+		let dummyError: Error = new Error(starkHttpServiceName + ": Error getting a SingleItemResponse");
```

Comment:
Could you add a comment to explain why we create this "dummyError" here? This way it will be clear for anyone who sees this code ;)

And perhaps we could rename this error to be more descriptive instead of "dummy..."